### ISSUE TITLE
PyPI Release

### ISSUE THREAD

**Issue by lucacorti** [afrancis13/celery-bungiesearch#5]
Any plan to make a PyPI release? 1.0.0 seems to be there and it is broken.

**Comment by afrancis13** [comment]
There have indeed been several updates since 1.0.0. I was planning on doing a PyPI release, but haven't got to it yet. I will try to get a new release out with some release notes this afternoon or evening (PST).

**Comment by afrancis13** [comment]
I just made the 1.1.0 release on PyPI. Definitely let me know if you issues, and if not, please close the issue. Thanks!

**Comment by lucacorti** [comment]
Thanks for the quick response and release @afrancis13.

Unfortunately, `pip install celery-bungiesearch` does not install the module. I only have the `celery_bungiesearch-1.1.0-py2.7.egg-info` directory in my venv `site-packages`.

```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "lib/python2.7/site-packages/django/core/management/__init__.py", line 312, in execute
    django.setup()
  File "lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "lib/python2.7/site-packages/django/apps/config.py", line 198, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "items/models.py", line 24, in <module>
    class Item(models.Model):
  File "lib/python2.7/site-packages/django/db/models/base.py", line 189, in __new__
    new_class.add_to_class(obj_name, obj)
  File "lib/python2.7/site-packages/django/db/models/base.py", line 324, in add_to_class
    value.contribute_to_class(cls, name)
  File "lib/python2.7/site-packages/bungiesearch/managers.py", line 39, in contribute_to_class
    self.signal_processor = get_signal_processor()
  File "lib/python2.7/site-packages/bungiesearch/signals.py", line 14, in get_signal_processor
    signal_module = import_module('.'.join(signal_path[:-1]))
  File "/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named celery_bungiesearch.signals
```

**Comment by lucacorti** [comment]
`pip` also reports no installed `.py` files.

```
$ pip show -f celery-bungiesearch
---
Metadata-Version: 1.1
Name: celery-bungiesearch
Version: 1.1.0
Summary: Celery signal processor for Bungiesearch
Home-page: https://github.com/afrancis13/celery-bungiesearch
Author: Alex Francis
Author-email: afrancis@berkeley.edu
License: MIT
Location: /Users/luca/Documents/snap-django/venv/lib/python2.7/site-packages
Requires: bungiesearch, celery, Django
Files:
  celery_bungiesearch-1.1.0-py2.7.egg-info
  celery_bungiesearch-1.1.0-py2.7.egg-info/PKG-INFO
  celery_bungiesearch-1.1.0-py2.7.egg-info/SOURCES.txt
  celery_bungiesearch-1.1.0-py2.7.egg-info/dependency_links.txt
  celery_bungiesearch-1.1.0-py2.7.egg-info/pbr.json
  celery_bungiesearch-1.1.0-py2.7.egg-info/requires.txt
  celery_bungiesearch-1.1.0-py2.7.egg-info/top_level.txt
```

**Comment by lucacorti** [comment]
Seems to be fixed by 1.1.4. Thanks @afrancis13. Closing
